apis/python/tests/test_experiment_query.py

```diff
@@ -547,6 +577,72 @@ def test_experiment_query_obsp_varp(soma_experiment):
             .concat()
         )
 
+        assert (
+            query.obsm("baz").tables().concat()
+            == soma_experiment.ms["RNA"]
+            .obsm["baz"]
+            .read((obs_slice, range(50)))
```

Comment:
magic number 50 is in a few places.  Would be good to at least use a constant variable (e.g., N_FEAT or some such)

Reply:
Good point, let me add it to a constant.